src/BreweryScrapper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class BreweryScrapper
+{
+    protected Crawler $sections;
+    private array $casparyData = [];
+
+    public function casparyScrapper(): void
+    {
+        $url = "https://www.caspary.com/en/references.html";
+        $html = file_get_contents($url);
+        $crawler = new Crawler($html);
+
+        $this->walker = $crawler->filterXPath("//*[@id='referenzmenu']");
+        $fetchedCountry = $this->walker->filterXPath("//span[@class='col-sm-12 title']");
+        $item = $this->walker->filterXPath("//span");
+
+        $countries = [];
+        $fetchedRecords = [];
+        $countryName = "";
+
+        for ($pointerPosition = 0; $pointerPosition < $fetchedCountry->count(); $pointerPosition++) {
+            $countries[] = $fetchedCountry->eq($pointerPosition)->text();
+        }
+
+        for ($pointerPosition = 0; $pointerPosition < $item->count(); $pointerPosition++) {
+            if ($item->eq($pointerPosition)->text() !== "") {
+                $fetchedRecords[] = $item->eq($pointerPosition)->text();
+            }
+        }
+
+        for ($pointerPosition = 0; $pointerPosition < count($fetchedRecords) - 1; $pointerPosition += 2) {
+            if (in_array($fetchedRecords[$pointerPosition], $countries, true)) {
+                $countryName = $fetchedRecords[$pointerPosition];
+                $add = [
+                    "name" => $fetchedRecords[$pointerPosition + 1],
+                    "city" => $fetchedRecords[$pointerPosition + 2],
+                    "country" => $countryName,
+                    "provider" => "Caspary",
+                ];
+                $this->casparyData[] = $add;
+                $pointerPosition++;
+            } else {
+                $add = [
+                    "name" => $fetchedRecords[$pointerPosition],
+                    "city" => $fetchedRecords[$pointerPosition + 1],
+                    "country" => $countryName,
+                    "provider" => "Caspary",
+                ];
+                $this->casparyData[] = $add;
+            }
+        }
```

Comment:
Is this loop working as expected? 
there is `$pointerPosition += 2` but in if statement there is `$pointerPosition++;`

So at the end of each iteration `pointerPosition` will be +2 or +3 when if statement was true?

Reply:
Statement is true when pointer meets first brewery of currently choosed city. In this situation I want to "jump" above 2 records (as in if statement) and this one extra, which stands for country record

The collected data has an irregular pattern, so the `$pointerPosition` must change the number of increments in specific cases

Answering your question: when "if" statement is true, `$pointerPosition` is +3. else it's +2